### PR TITLE
Fix wrong conflict resolution in FIP-0083 update

### PR DESCRIPTION
**Pull request by aarshkshah1992**
We accidentally resolved the conflict wrongly before merging https://github.com/filecoin-project/FIPs/pull/968.
This PR fixes that.

### CODE REVIEW

**Review by TippyFlitsUK — APPROVED** [review]
Apologies for the initial error @aarshkshah1992!!

**Review by jsoares — APPROVED** [review]
fix in accordance with b01a6edd94ed458d195fbc56da7b41e1712ce533, looks ok

File: FIPS/fip-0083.md [hunk 1]
```diff
@@ -62,11 +62,16 @@ This event is emitted when the balance of a verifier is updated in the Verified
...
+| flags              | key        | value                               |
+|--------------------|------------| ----------------------------------- |
+| Index Key + Value  | "$type"    | "verifier-balance" (string)         |
+| Index Key + Value  | "verifier" | <VERIFIER_ACTOR_ID> (int)           |
+| Index Key + Value  | "client"   | <DATACAP_HOLDER_ACTOR_ID> (optional int) (only present when datacap allocated to a client) |
+| Index Key          | "balance"  | <VERIFIER_DATACAP_BALANCE> (bigint) |
+
+The "bigint" CBOR encoding format used for the "balance" field is the same as is used for encoding bigints on the Filecoin chain: a byte
+array representing a big-endian unsigned integer, compatible with the Golang big.Int byte representation, with a 0x00 (positive)
+or 0x01 (negative) prefix; with a zero-length array representing a value of 0.
```
Comment (anorth) [inline FIPS/fip-0083.md:74]: This paragraph was added at line 53 and should not be duplicated here.

Comment (rvagg) [inline FIPS/fip-0083.md:74]:
```suggestion
| flags              | key        | value                                                                                      |
|--------------------|------------| ------------------------------------------------------------------------------------------ |
| Index Key + Value  | "$type"    | "verifier-balance" (string)                                                                |
| Index Key + Value  | "verifier" | <VERIFIER_ACTOR_ID> (int)                                                                  |
| Index Key + Value  | "client"   | <DATACAP_HOLDER_ACTOR_ID> (optional int) (only present when datacap allocated to a client) |
| Index Key          | "balance"  | <VERIFIER_DATACAP_BALANCE> (bigint)                                                        |
```

Sorry, I couldn't do nice stacked commits from a fork, so I opted for the way that needed reconciliation but I forgot to call that out. My fault.

This change is what it should be (was, before I rebased on `master` to make the diff minimal).

**Comment by willscott** [comment]
Is this expected to land for NV22?

**Comment by rvagg** [comment]
yes, it's in actors v13.0.0 @ https://github.com/filecoin-project/builtin-actors/blob/v13.0.0/actors/verifreg/src/emit.rs#L26 and Lotus v1.26.0, although it's not on calibnet, there's a minor upgrade there just for this and the other verifreg changes, see notes in v1.26.1 for timing of that upgrade: https://github.com/filecoin-project/lotus/releases/tag/v1.26.1

@aarshkshah1992 can you clean up the conflict here and get it ready to land this PR?

**Comment by aarshkshah1992** [comment]
Closing in favour of https://github.com/filecoin-project/FIPs/pull/980

**Pull request closed without merging**